### PR TITLE
Fixes a critical bug with the weather subsystem

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
fix: Ash storms can now occur again.
/:cl:

The syntax of `addtimer()` meant that any number (read: any z-level's number) was being assumed as telling the timer that it was unique, leading to ash storms never occurring past the first one.

This is tested and confirmed to work.

### CODE REVIEW

File: code/controllers/subsystem/weather.dm [hunk 1]
```diff
@@ -30,7 +30,7 @@ var/datum/subsystem/weather/SSweather
 		var/datum/weather/W = pickweight(possible_weather_for_this_z)
 		run_weather(W.name)
 		eligible_zlevels -= Z
-		addtimer(src, "make_z_eligible", rand(3000, 6000) + W.weather_duration_upper, Z) //Around 5-10 minutes between weathers
+		addtimer(src, "make_z_eligible", rand(3000, 6000) + W.weather_duration_upper, TRUE, Z) //Around 5-10 minutes between weathers
```
Comment (ChangelingRain) [inline code/controllers/subsystem/weather.dm]: wouldn't TRUE also make it unique? this should be FALSE

Comment (coiax) [inline code/controllers/subsystem/weather.dm]: yeah, it's addtimer(object, "proctocall", time2wait, uniquebool, ARGS? ...)

Comment (Xhuis) [inline code/controllers/subsystem/weather.dm]: I intended it to be unique. If I read the code correctly, not having it be unique could result in several ash storms at once and, while that'd be hilarious, it probably isn't good.

**Comment by Xhuis** [comment]
Merge this

**Pull request merged**